deploy/awstools/awstools.py

```diff
@@ -116,12 +72,16 @@ def launch_run_instances(instancetype, count, fsimclustertag, instancemarket, sp
     vpcfilter = [{'Name':'tag:Name', 'Values': [vpcname]}]
     firesimvpc = list(ec2.vpcs.filter(Filters=vpcfilter))
     subnets = list(firesimvpc[0].subnets.filter())
+    random.shuffle(subnets)
```

Comment:
Can you remove the randomization?

Reply:
If randomization is problematic for run instances, then we can add a `randomize=false` optional argument.  Otherwise, it seems somewhat useful to not have all spot requests for build instances skew towards earlier subnets, even if they initially succeed.

Reply:
Yeah, let's do that.